### PR TITLE
Adding a simple APLT Demo

### PR DESCRIPTION
**Pull request by JoeMoCode**
Tested using Alexa Hosted Custom skill and an Echo Dot with Clock.

*Description of changes:*
Adds an APLT demo which will run on an Echo Dot with clock.

### New Feature Demo
*Feature being demonstrated:*
APLT

### CODE REVIEW

File: feature-demos/apl/skill-demo-aplt-text/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Catalog Entities Demo
```
Comment (franklin-lobb) [inline feature-demos/apl/skill-demo-aplt-text/README.md]: wrong demo title

File: feature-demos/apl/skill-demo-aplt-text/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+## What You Will Need
+*  [Amazon Developer Account](http://developer.amazon.com/alexa)
+*  The sample code on [GitHub](https://github.com/alexa/alexa-cookbook/tree/master/feature-demos/skill-demo-aplt-text).
+
+## Setting Up the Demo
+If you use Alexa hosted, follow the steps:
+
+1. Create a new Alexa Hosted skill using Node.JS
+2. Copy paste the file models/en-US.json into the json editor of the interaction model. Build this.
```
Comment (franklin-lobb) [inline feature-demos/apl/skill-demo-aplt-text/README.md]: also needs the Alexa Presentation Language interface enabled on the Interfaces page.

**Comment by franklin-lobb** [comment]
@JoeMoCode - I tried this out - nice!  found two little things in the read me.  All it needs is a skill.json and you could git-clone and ask-deploy!  :)

**Comment by JoeMoCode** [comment]
Good Calls, Franklin! Addressed the comments.

**Pull request merged**